src/sonic-bgpcfgd/bgpcfgd/main.py

```diff
@@ -81,6 +80,10 @@ def do_work():
         SRv6Mgr(common_objs, "CONFIG_DB", "SRV6_MY_SIDS"),
         SRv6Mgr(common_objs, "CONFIG_DB", "SRV6_MY_LOCATORS")
     ]
+    if device_info.is_chassis():
+        managers.append(BGPPeerMgrBase(common_objs, "CONFIG_DB", "BGP_VOQ_CHASSIS_NEIGHBOR", "voq_chassis", False))
+    else:
```

Comment:
Wouldn't this apply on T0/T1 as well?

Reply:
> Wouldn't this apply on pizzabox as well?

No. In T0 and T1, the "BGP_VOQ_CHASSIS_NEIGHBOR" is not present in the CONFIG_DB, so bgpd will not get the config notification from redis.  If you look at the old before my change, this line of code was there.